src/main/java/step2/domain/LottoNumber.java

```diff
@@ -0,0 +1,23 @@
+package step2.domain;
+
+public class LottoNumber {
+    private final int number;
+    private static final String LOTTO_NUMBER_RANGE_ERROR_MESSAGE = "로또 당첨번호는 1과 45 사이의 정수이어야 합니다.";
+    private static final int MIN_OF_LOTTO_NUMBER = 1;
+    private static final int MAX_OF_LOTTO_NUMBER = 45;
```

Comment:
```suggestion
    private static final int MIN = 1;
    private static final int MAX = 45;
    private static final String LOTTO_NUMBER_RANGE_ERROR_MESSAGE = "로또 당첨번호는 " + MIN + "과 " + MAX + " 사이의 정수이어야 합니다.";
```

메시지에서 상수들을 사용한다면 상수가 변경 됐을 때 메시지를 신경쓰지 않으셔도 됩니다 :)

또한 LottoNumber. MIN_OF_LOTTO_NUMBER에서 LottoNumber라는 이름이 중복되고 있네요!
Integer.MAX_INTEGER_VALUE vs Integer.MAX_VALUE 의 차이를 생각 했을 때 LottoNumber.MIN이 더 적절할 것 같은데 어떻게 생각하시나요?

Reply:
아하! 클래스 명과, 변수 명을 따로 생각하다 보니 이러한 문제가 발생한 것 같습니다.
확인 감사합니다!